exchanges/fundingrate/fundingrate_types.go

```diff
@@ -0,0 +1,70 @@
+package fundingrate
```

Comment:
could rn package to funding to reduce stutter

Reply:
I think funding is too vague for it to be solely about funding rates. funding package for backtester is about managing and maintaining funds for a run as a counter-example